src/main/java/com/mitrais/cdc/bloggatewayapplicationv1/controller/UserController.java

```diff
@@ -118,7 +119,7 @@
             log.error(e.getMessage(), e);
 
         }
-        return ResponseEntity.ok(new Utility("Sending email to reset password was failed", user).getResponseData());
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(new Utility("Sending email to reset password was failed", user).getResponseData());
```

Comment:
I see you're now returning error code which is OK. Have looked into the exception handler?? In that way, you don't even need to worry about returning errors, the handler will do it for you (as long as you code it correctly)